README.md

```diff
@@ -442,6 +442,16 @@ You can download and install `moodycamel::ConcurrentQueue` using the [vcpkg](htt
 	
 The `moodycamel::ConcurrentQueue` port in vcpkg is kept up to date by Microsoft team members and community contributors. If the version is out of date, please [create an issue or pull request](https://github.com/Microsoft/vcpkg) on the vcpkg repository.
 
+## Using apt on Debian 11 (Linux)
+You can download and install 'moodycamel::ConcurrentQueue' in one step using aptitutde:

```

Comment:
Please include who maintains this package, and what to do if it's out of date (similar to the `vcpkg` instructions above).

Reply:
Also, "aptitude" is misspelled with an extra 't'.